ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.create-run.tsx

```diff
@@ -987,7 +1017,83 @@ const CreateRunPage = () => {
+                          </FormControl>
+                          <FormDescription>
+                            The path to the rules file which is resolved from
+                            the configured configuration source. Note: in case
+                            this field is left empty, the default path of the
+                            config file provider has to include a rules file,
+                            otherwise the Evaluator job will fail.
+                          </FormDescription>
+                          <FormMessage />
+                        </FormItem>
+                      )}
+                    />
+                    <FormField
+                      control={form.control}
+                      name='jobConfigs.evaluator.licenseClassificationsFile'
+                      render={({ field }) => (
+                        <FormItem className='pt-4'>
+                          <FormLabel>License classifications</FormLabel>
+                          <FormControl>
+                            <Input {...field} />
+                          </FormControl>
+                          <FormDescription>
+                            The path to the license classifications file which
+                            is resolved from the configured configuration
+                            source.
+                          </FormDescription>
+                          <FormMessage />
+                        </FormItem>
+                      )}
+                    />
+                    <FormField
+                      control={form.control}
+                      name='jobConfigs.evaluator.copyrightGarbageFile'
+                      render={({ field }) => (
+                        <FormItem className='pt-4'>
+                          <FormLabel>Copyright garbage</FormLabel>
+                          <FormControl>
+                            <Input {...field} />
+                          </FormControl>
+                          <FormDescription>
+                            The path to the copyright garbage file which is
+                            resolved from the configured configuration source.
+                          </FormDescription>
+                          <FormMessage />
+                        </FormItem>
+                      )}
+                    />
+                    <FormField
+                      control={form.control}
+                      name='jobConfigs.evaluator.resolutionsFile'
+                      render={({ field }) => (
+                        <FormItem className='pt-4'>
+                          <FormLabel>Resolutions</FormLabel>
+                          <FormControl>
+                            <Input {...field} />
+                          </FormControl>
+                          <FormDescription>
+                            The path to the resolutions file which is resolved
+                            from the configured configuration source.
```

Comment:
Esp. in this description there's a few word duplications: "resolutions" / "resolved", "configured" / "configuration". I propose to just say: "The path to the resolutions file to get from the configuration provider."

Similar above.